backend/flask_start.sh

```diff
@@ -0,0 +1,4 @@
+set -o errexit
+set -o nounset
+cd /app
+gunicorn -b :5000 app:flask_app --log-level DEBUG
```

Comment:
should de default logging level be "DEBUG", here?

Reply:
This logging level was left from the original Dockerfile. I'll probably should change it